src/renderers/dom/fiber/ReactDOMFiber.js

```diff
@@ -197,6 +212,18 @@ var DOMRenderer = ReactFiberReconciler({
     return true;
   },
 
+  commitInitialEffects(
+    domElement : Instance,
+    type : string,
+    newProps : Props,
+    rootContainerInstance : Container,
+    internalInstanceHandle : Object,
+  ) : void {
+    if (shouldAutoFocusHostComponent(type, newProps)) {
+      (domElement : any).focus();
+    }
```

Comment:
Makes sense. Let's go ahead and call that method `commitMount` then. Initial is redundant.